src/devices/src/legacy/serial.rs

```diff
@@ -102,12 +139,14 @@ pub enum Error {
 
 #[cfg(test)]
 mod tests {
-    use crate::EventFdTrigger;
-
     use super::SerialWrapper;
+    use crate::legacy::EventFdTrigger;
 
     use std::io::sink;
 
+    #[cfg(target_arch = "aarch64")]
+    use vm_device::{bus::MmioAddress, MutDeviceMmio};
+    #[cfg(target_arch = "x86_64")]
     use vm_device::{bus::PioAddress, MutDevicePio};
     use vm_superio::Serial;
```

Comment:
We can use "use super::*" and avoid reimporting things explicitly here.

Reply:
Added `use super::*`.